src/BenchmarksApps/MapAction/Program.cs

```diff
@@ -20,8 +20,12 @@
                 Todo EchoTodo([FromBody] Todo todo) => todo;
                 endpoints.MapPost("/EchoTodo", (Func<Todo, Todo>)EchoTodo);
                 // Intentionally disable anti-forgery for this endpoint so we can evaluate just the form-processing portions
-                endpoints.MapPost("/EchoTodoForm", ([FromForm] Todo todo) => todo)
```

Comment:
Are we going to add a form binding benchmark later?

Reply:
Yeah, as a different benchmark. I think when all is said and done we'll have the following scenarios:

- POST to IFormCollection
- POST to [FromForm] with complex type
- POST to IFormCollection with anti-forgery enabled

The should allow us to measure each component of form-processing somewhat independently:

- Reading the form body into an IFormCollection
- Running the anti-forgery checks
- Binding from the form body to a complex type